src/components/Chat.js

```diff
@@ -1,3 +1,7 @@
+//useRef, io, link, edit, make_room, chat_bubble, and rocket are defined but never used
```

Comment:
`io` is used in line 2 of the original file, line 6 of the current commit.

Reply:
Line 6 shows that it is imported in the current commit, however, it is still never used